Gemfile

```diff
@@ -88,6 +88,7 @@ gem 'rails-assets-jquery',   '1.11.1' # Should be kept in sync with jquery-rails
 gem 'js_image_paths',        '0.0.1'
 gem 'js-routes',             '0.9.9'
 gem 'rails-assets-punycode', '1.3.2'
+gem 'rails-assets-twemoji'
```

Comment:
Let's keep that change separate, since I disagree with it while agreeing about changing the database encoding.